### ISSUE TITLE
[JMM Desktop] Nyaa torrent search is not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#423]
```
What steps will reproduce the problem?
1. Go to Downloads section
2. Go to Search Torrents tab
3. Make sure Nyaa is enabled as a search engine
4. Search for some anime (I used Joshiraku, but it doesn't matter)

What is the expected output? What do you see instead?
I see search results from TokyoTosho, but nothing from Nyaa, even though it is 
enabled. Going to the site directly clearly shows results for my search term, 
so something has probably changed. I occasionally see garbled xml text as a 
search result from Nyaa, which further confirms my suspicion.

What version of the product are you using? On what operating system?
JMM Desktop 3.1.28.0
Windows 7

Please provide any additional information below.
Connected to remote JMM Server 3.1.31.0 on separate machine (in same LAN). 
Windows Home Server 2011
```

Original issue reported on code.google.com by `jun.esmo...@gmail.com` on 17 Apr 2013 at 3:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 28 Apr 2013 at 11:16
- Changed state: **Accepted**
- Added labels: **Priority-Critical**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.32
```

Original comment by `werndly...@gmail.com` on 1 May 2013 at 1:54
- Changed state: **Fixed**
